viewing_party/party.py

```diff
@@ -1,23 +1,142 @@
+

```

Comment:
We should take care to not add in unnecessary changes to a commit so that our project stays neat. Looks like an unneeded blank line was added on line 1 here.